auth0/provider_test.go

```diff
@@ -154,9 +154,9 @@ func TestProvider_configValidation(t *testing.T) {
 		})
 	}
 
-	for _, e := range originalEnviroment {
+	for _, e := range originalEnvironment {
 		environmentPair := strings.Split(e, "=")
-		os.Setenv(environmentPair[0], environmentPair[1])
+		_ = os.Setenv(environmentPair[0], environmentPair[1])
```

Comment:
**Questions (non-blocking):** Assuming this is intentional, what is the functional purpose for this? And in the spirit of not ignoring errors, shouldn't we surface this one?

Reply:
This is intentional indeed, considering that here we just set again what we had before in the `os.Environ()` there's no way this can fail, so we're saving a few lines of code. 🤷🏻  

The `os.Setenv` only fails if we provide invalid input to it and even let's say hypothetically this fails, we'll see it immediately in the tests that run afterwards and depend on those env vars.